cms/djangoapps/contentstore/views/course.py

```diff
@@ -536,41 +535,6 @@ def _get_rerun_link_for_item(course_key):
     return reverse_course_url('course_rerun_handler', course_key)
 
 
-def _deprecated_blocks_info(course_module, deprecated_block_types):
```

Comment:
hmm.. is this code no longer needed?  I'm not aware of the need to remove this code as well?  Is there a separate ticket for that?